### PR TITLE
Update Blog Contribute page.

### PR DESCRIPTION
**Pull request by aFinchy**
Signed-off-by: Amy Finch <amy@finchstudio.ca>

## Change summary

Created an updated blog contributing page in an effort to help migrate the efforts to the public and LF responsibility. 

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [Style Guide](https://o3de.org/docs/contributing/to-docs/style-guide/quick-reference)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Comment by aFinchy** [comment]
image

**Comment by chanmosq** [comment]
Per discussion offline with @aFinchy -
At this time, there are two PRs submitting the Blogs Process document to the [`o3de/community` repo](https://github.com/o3de/community/pull/182/) and to the Contributing Guide of the O3DE Website (this one). However, the Blogs Process documentation should live only in one location for ease of maintainability and to avoid two documents becoming out-of-sync.

This document more closely fits into the `o3de/community` repo because publishing a blog is primarily an action taken by a member of the O3DE community who coordinates with the blog author. Then, this page should be repurposed to direct folks to that document.

**Comment by aFinchy** [comment]
@chanmosq 

How would this sound on the page? 

If you're interested in writing or contributing to blogs about the Open 3D Engine, you can find a wealth of resources, including a detailed guide, templates, examples, and more, by [visiting the document located here](https://github.com/o3de/community/tree/main/markdown)

https://github.com/o3de/community/tree/main/markdown.

**Comment by aFinchy** [comment]
image

**Review by FiniteStateGit — APPROVED** [review]

File: content/docs/contributing/to-docs/blog-posts.md
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:12]: The link doesn't point to a valid resource on GitHub.

File: content/docs/contributing/to-docs/blog-posts.md
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:9]: I would move this heading after any introductory information on blogs and rename it to something like "Submit a blog" or "Blog submission process".  Headings should use sentence casing.

File: content/docs/contributing/to-docs/blog-posts.md
Comment (FiniteStateGit) [inline content/docs/contributing/to-docs/blog-posts.md:11]: I wasn't able to view the markdown and process guides, did the content that you propose removing from the current page (e.g. _What makes a good O3DE blog post_, _Do's and Dont's_) move to a guide?

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/contributing/to-docs/blog-posts.md
Comment (chanmosq) [reply content/docs/contributing/to-docs/blog-posts.md:12]: The link should work after this PR is merged: https://github.com/o3de/community/pull/187
Preview of doc in feature-branch: https://github.com/o3de/community/blob/Blog-Process/process-docs-and-markdown/blog-process.md

File: content/docs/contributing/to-docs/blog-posts.md
Comment (chanmosq) [reply content/docs/contributing/to-docs/blog-posts.md:9]: I've added a comment to add that section to this guide: https://github.com/o3de/community/pull/187/files

**Review by chanmosq — APPROVED** [review]

**Review by willihay — APPROVED** [review]
Approved pending merge of https://github.com/o3de/community/pull/187

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**
